src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, Dispatch, SetStateAction } from 'react';
+
+import {
+  EmptyState,
+  EmptyStateActions,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+  SearchInput,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { Link } from 'react-router-dom';
+
+import BlueprintCard from './BlueprintCard';
+
+import { BlueprintItem } from '../../store/imageBuilderApi';
+import { resolveRelPath } from '../../Utilities/path';
+
+interface blueprintProps {
+  blueprints: BlueprintItem[] | undefined;
+  selectedBlueprint: string;
+  setSelectedBlueprint: Dispatch<SetStateAction<string>>;
+}
+const BlueprintsSidebar = (props: blueprintProps) => {
+  const [blueprintFilter, setBlueprintFilter] = useState('');
+
+  const onChange = (value: string) => {
+    setBlueprintFilter(value);
+  };
+
+  const blueprintsEmpty = (
+    <EmptyState variant="sm">
+      <EmptyStateHeader
+        titleText="No blueprints yet"
+        headingLevel="h4"
+        icon={<EmptyStateIcon icon={PlusCircleIcon} />}
+      />
+      <EmptyStateBody>To get started, create a blueprint.</EmptyStateBody>
+      <EmptyStateFooter>
+        <EmptyStateActions>
+          <Link
+            to={resolveRelPath('blueprintwizard')}
+            className="pf-c-button pf-m-primary pf-u-mr-md"
+            data-testid="create-blueprint-action"
+          >
+            Create
+          </Link>
+        </EmptyStateActions>
+      </EmptyStateFooter>
+    </EmptyState>
+  );
+
+  return props.blueprints?.length ? (
+    <>
+      <SearchInput
+        placeholder="Search by name or description"
+        value={blueprintFilter}
+        onChange={(_event, value) => onChange(value)}
+        onClear={() => onChange('')}
+        className="pf-v5-u-mb-md"
```

Comment:
What do we need this for? I would prefer to not need to specify any CSS classes and rely entirely on Patternfly's props.

Reply:
leftovers from previous PR

Reply:
I see. Can we remove it?

Reply:
@jkozol can you advise?

Reply:
removed